module-data/src/main/java/org/lindeck/data/model/Deck.java

```diff
@@ -0,0 +1,46 @@
+package org.lindeck.data.model;
+
+import lombok.Data;
+import org.hibernate.annotations.GenericGenerator;
+import org.lindeck.data.model.cards.AbstractCard;
+
+import javax.persistence.*;
+import java.util.List;
+import java.util.UUID;
+
+@Entity
+@Data
+@Table(name = "decks")
+public class Deck {
+    /**
+     * Decks can be created locally, so it's good case to user UUID
+     */
+    @Id
+    @GeneratedValue(generator = "UUID")
+    @GenericGenerator(name = "UUID",
+                      strategy = "org.hibernate.id.UUIDGenerator")
+    private UUID id;
+
```

Comment:
Decks can be created localy. -> So when they did, Frontend created UUID for the deck. So u do not need to Generate UUID on the server again.